modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line
+      format: raw   # add this line
+controlPlane:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line <1>
+      format: raw   # add this line <2>
 ----
 
+<1> Sparse - Setting this option to `false` enables preallocation of disks. Default: `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
```

Comment:
You could remove "Sparse:" from this line, because the call out numbering shows which option is being referred to.